go/vt/vttablet/tabletserver/throttle/throttler.go

```diff
@@ -656,7 +655,14 @@ func (throttler *Throttler) Operate(ctx context.Context) {
 	throttledAppsTicker := addTicker(throttler.throttledAppsSnapshotInterval)
 	recentCheckTicker := addTicker(time.Second)
 
+	wg.Add(1)
 	go func() {
+		defer wg.Done()
+		defer throttler.aggregatedMetrics.Flush()
+		defer throttler.recentApps.Flush()
+		defer throttler.nonLowPriorityAppRequestsThrottled.Flush()
```

Comment:
Any reason not to use a single deferred function, at least for the flushes? It's fine as-is, just curious.

Reply:
Consolidated.